freqtrade/rpc/rpc_manager.py

```diff
@@ -66,7 +78,11 @@ def send_msg(self, msg: Dict[str, Any]) -> None:
             'status': 'stopping bot'
         }
         """
-        logger.info('Sending rpc message: %s', msg)
+        # Removed actually showing the message because the logs would be
+        # completely spammed of the json dataframe
+        logger.info('Sending rpc message of type: %s', msg.get('type'))
+        # Log actual message in debug?
+        # logger.debug(msg)
```

Comment:
this can (and probably should) become conditional (not logging for dataframes).

i think the remainder of the changes in this file should be reverted - as they're all outdated.